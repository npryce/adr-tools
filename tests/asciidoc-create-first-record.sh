echo "adoc" > .adr-file-type
adr new The First Decision
cat doc/adr/0001-the-first-decision.adoc
