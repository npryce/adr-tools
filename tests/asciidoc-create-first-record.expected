echo "adoc" > .adr-file-type
adr new The First Decision
doc/adr/0001-the-first-decision.adoc
cat doc/adr/0001-the-first-decision.adoc
= 1. The First Decision

Date: 1992-01-12

== Status

Accepted

== Context

The issue motivating this decision, and any context that influences or constrains the decision.

== Decision

The change that we're proposing or have agreed to implement.

== Consequences

What becomes easier or more difficult to do and any risks introduced by the change that will need to be mitigated.
